linkerd/app/inbound/src/http/tests.rs

```diff
@@ -37,9 +37,14 @@ fn build_server<I>(
 where
     I: io::AsyncRead + io::AsyncWrite + io::PeerAddr + Send + Unpin + 'static,
 {
-    let Config { proxy, .. } = cfg.clone();
+    // Mocks to_tcp_connect.
+    let connect = svc::stack(connect)
+        .push_map_target(|t: TcpEndpoint| Remote(ServerAddr(([127, 0, 0, 1], t.param()).into())))
+        .push_connect_timeout(cfg.proxy.connect.timeout)
+        .into_inner();
```

Comment:
This is simple enough that I'm comfortable duplicating what's in `to_tcp_connect` -- I don't think it's worth having this all factored out for the `push_connect_timeout` (the map_target should go away in an upcoming change).